sensor/scripts/enviroplushat.py

```diff
@@ -36,17 +44,46 @@ def get_readings(self, sensor):
         current_oxidising = current_gas.oxidising / 1000
         current_reducing = current_gas.reducing / 1000
         current_nh3 = current_gas.nh3 / 1000
-        return [
-            {
-                'measurement': 'balena-sense',
-                'fields': {
-                    'temperature': float(current_temperature),
-                    'pressure': float(current_pressure),
-                    'humidity': float(current_humidity),
-                    'light': float(current_lux),
-                    'oxidising': float(current_oxidising),
-                    'reducing': float(current_reducing),
-                    'nh3': float(current_nh3)
+
+        try:
+            pm_data = self.pms5003.read()
+        except pmsReadTimeoutError:
+            logging.warn("Failed to read PMS5003")
+        else:
+            pm1 = pm_data.pm_ug_per_m3(1.0)
+            pm2 = pm_data.pm_ug_per_m3(2.5)
+            pm10= pm_data.pm_ug_per_m3(10.0)
+
+        if "ENVIRO_PLUS_PM" in os.environ:
```

Comment:
Not a huge fan of having this stuff all repeated, but could think of a quick cleaner way yet